terraform/aws/security/aws-ssm-document-logging-issues.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+- id: aws-ssm-document-logging-issues
+  patterns:
+  - pattern-inside: |
+      resource "aws_ssm_document" $ANYTHING {
+        ...
+      }
+  - pattern: content = "$STATEMENT"
+  - metavariable-pattern:
+      metavariable: $STATEMENT

```

Comment:
It looks like this might be missing the language field so that it scans JSON or YAML.

To do two languages with `metavariable-pattern`, you may have to break this up into two `patterns` clauses, one with `pattern: content="$STATEMENT"` and `metavariable-pattern` for JSON, and a second one with `pattern: content="$STATEMENT"` and `metavariable-pattern` for YAML.

Reply:
Will change it accordingly